tests/fixtures/make-breach-with-emails.js

```diff
@@ -1,60 +1,64 @@
 "use strict";
 
+const arg = require("arg");
+
 require("dotenv").load();
 
 const DBUtils = require("../../db/utils");
 
 
-const extraTestEmail = process.argv[2];
+const args = arg({
+  "--createAMBreach": Boolean,
+  "--extraTestEmail": String,
+  "--help": Boolean,
+});
+
+if (args["--help"]) {
+  console.log("Usage: node make-breach-with-emails.js [--createAMBreach] [--extraTestEmail=<...>]");
+  console.log("--createAMBreach creates the 'AllMusic' test fixture breach.");
+  console.log("--extraTestEmail adds the supplied test email address and includes it in the LinkedIn, Adobe, and AllMusic breaches.");
+  // We can `process.exit()` here since it's a CLI script.
+  // eslint-disable-next-line no-process-exit
+  process.exit();
+}
 
 const sampleBreaches = [
   {
-    name: "Linkedin",
-    meta: {
-      date: "5 May 2012",
-      dataClasses: "Email addresses, Passwords",
-      acCount: "164,611,595",
-    },
+    name: "LinkedIn",
     emails: [ "test1@test.com", "test2@test.com" ],
   },
   {
     name: "Adobe",
-    meta: {
-      date: "4 October 2013",
-      dataClasses: "Email addresses, Password hints, Passwords, Usernames",
-      acCount: "152,445,165",
-    },
     emails: [ "test2@test.com", "test3@test.com" ],
   },
   {
     name: "AllMusic",
-    meta: {
-      date: "6 December 2015",
-      dataClasses: "Email addresses, IP addresses, Passwords, Usernames, Website activity",
-      acCount: "1,436,486",
-    },
     emails: [ "test3@test.com", "test1@test.com" ],
   },
 ];
 
 (async () => {
+  if (args["--createAMBreach"]) {
+    await DBUtils.createBreach("AllMusic", {
+      Name: "AllMusic",
+      BreachDate: "2015-012-06",
+      DataClasses: ["Email addresses", "IP addresses", "Passwords", "Usernames", "Website activity"],
+      PwnCount: 1436486,
+    });
+  }
   for (const sB of sampleBreaches) {
-    await DBUtils.createBreach(sB.name, sB.meta);
+    await DBUtils.getBreachByName(sB.name);
```

Comment:
Why are we getting a breach and not doing anything with it?

Reply:
I'm not sure I understand what the purpose of this code is anymore. Why remove the meta data from the other sample breaches? Why try to add the extra emails for LinkedIn/Adobe when they may not exist? (We only have an option to add the all music breach?)

Reply:
Hmm ... yeah I'm sorta mangling these scripts together. 😢 I should add a call to `load-breaches` from here, or maybe rename this to `add-breached-emails.js` ...

Basically, I'm changing this `make-breach-with-emails.js` to only add the breached emails. It adds the `test[1-3]@test.com` emails to the `LinkedIn`, `Adobe`, and `AllMusic` breaches as before, but it now relies on the new `load-breaches` script to create those breaches with the *real* data from HIBP.

I should also probably move the `--createAMBreach` code into the `load-breaches` script.

Let me try to clean this up a bit ...

Reply:
Okay, I un-mangled this logic and moved `load-breaches` and `add-breached-emails` into `scripts/` (per https://github.com/mozilla/blurts-server/pull/109#pullrequestreview-120666384) and updated the `README` with instructions on how to use them.

Needs a new r?